README.md

```diff
@@ -48,22 +52,23 @@ excluded from your proxy lookups.
 Running Tests Manually
 ======================
 
-The test server can be started using
+The test server can be started using this command:
 
     ./wpt serve
 
-This will start HTTP servers on two ports and a websockets server on
-one port. By default one web server starts on port 8000 and the other
+This will start HTTP servers on two ports and a WebSockets server on
+one port. By default, one web server starts on port 8000 and the other
 ports are randomly-chosen free ports. Tests must be loaded from the
-*first* HTTP server in the output. To change the ports, copy the
-`config.default.json` file to `config.json` and edit the new file,
-replacing the part that reads:
+*first* HTTP server in the output.
+
+To override the default ports, copy the `config.default.json` file to
+`config.json`, and replace the portion in the new file that reads
 
 ```
 "http": [8000, "auto"]
 ```
 
-to some port of your choice e.g.
+to some port of your choice – e.g.,
```

Comment:
Comma instead of – would seem more typical.

Reply:
yes, good point!